apps/github-cascading-app/package.json

```diff
@@ -25,6 +25,7 @@
     "@actions/github": "^6.0.0",
     "@angular-devkit/build-angular": "~18.2.0",
     "@angular-devkit/core": "~18.2.0",
+    "@angular-devkit/schematics": "~18.2.0",
```

Comment:
Why is it needed ?
It's a GH app without any deps on Angular.
Same remark for the others `@angular-devkit/*` and `@angular/*`
For `@angular-eslint/*` we can discuss (for the eslint migration) if we keep these rules or not when the package does not depends on `@angular/core`

Reply:
They are need from Nx point of view for the following reasons:
- They are imported (even with `import type`)
- They are part of exported file (even builders are exported)

`import type` dependencies should be provided in case the `skipLibCheck: false` (which enforce the provide of type dependencies) but as it is not runtime dependencies, they can be flagged as `optional`